osu.Game.Tests/Visual/UserInterface/TestSceneOsuMenu.cs

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using NUnit.Framework;
+using osu.Framework.Graphics;
+using osu.Framework.Testing;
+using osu.Game.Graphics.UserInterface;
+using osuTK.Input;
+
+namespace osu.Game.Tests.Visual.UserInterface
+{
+    public class TestSceneOsuMenu : OsuManualInputManagerTestScene
+    {
+        public override IReadOnlyList<Type> RequiredTypes => new[]
+        {
+            typeof(OsuMenu),
+            typeof(DrawableOsuMenuItem)
+        };
+
+        private OsuMenu menu;
+        private bool actionPeformed;
```

Comment:
insanely pedantic but just in case
```suggestion
        private bool actionPerformed;
```
(+ usages)